lib/polyglot.js

```diff
@@ -268,14 +268,18 @@
   // Does the dirty work. Creates a `RegExp` object for each
   // interpolation placeholder.
   function interpolate(phrase, options) {
-    for (var arg in options) {
+	Object.keys(options).forEach(function(arg) {
+    //for (var arg in options) {
       if (arg !== '_' && options.hasOwnProperty(arg)) {
+		// reaplcement value should be escaped to avoid special regex replacement symbols
+		// e.g.: $& or $1
+		var replacement = options[arg].replace ? options[arg].replace(/\$/g, '$$$$') : options[arg];
```

Comment:
What condition are you testing for by checking if "replace" exists? If you're checking that something is a string or a regex, we should check for that instead of duck-typing that `String#replace` exists.